libs/pages/clusters/src/lib/ui/page-clusters-create/step-kubeconfig/step-kubeconfig.tsx

```diff
@@ -0,0 +1,111 @@
+import { type FormEventHandler, useCallback } from 'react'
+import { useDropzone } from 'react-dropzone'
+import { useController, useFormContext } from 'react-hook-form'
+import { useNavigate, useParams } from 'react-router-dom'
+import { type ClusterKubeconfigData } from '@qovery/shared/interfaces'
+import { CLUSTERS_CREATION_GENERAL_URL, CLUSTERS_CREATION_URL, CLUSTERS_URL } from '@qovery/shared/routes'
+import { Button, Dropzone, Icon, IconAwesomeEnum } from '@qovery/shared/ui'
+
+export interface StepKubeconfigProps {
+  onSubmit: FormEventHandler<HTMLFormElement>
+}
+
+export function StepKubeconfig({ onSubmit }: StepKubeconfigProps) {
+  const navigate = useNavigate()
+  const { organizationId = '' } = useParams()
+  const { control, formState } = useFormContext<ClusterKubeconfigData>()
+  const { field: fileName } = useController<ClusterKubeconfigData>({
+    name: 'file_name',
+    control,
+    rules: { required: true },
+  })
+  const { field: fileContent } = useController<ClusterKubeconfigData>({
+    name: 'file_content',
+    control,
+    rules: { required: true },
+  })
+  const { field: fileSize } = useController<ClusterKubeconfigData>({
+    name: 'file_size',
+    control,
+    rules: { required: true },
+  })
+
+  const onDrop = useCallback((acceptedFiles: File[]) => {
```

Comment:
The callback isn't necessary?

Reply:
Not sure to understand this comment.
I've use `useCallback` because that's the [lib usage](https://react-dropzone.js.org/#usage)

Reply:
Yes I see the documentation but on our case we can avoid the `useCallback` 

``` 
useDropzone({
    maxFiles: 1,
    onDrop: (acceptedFiles) => {}
    },
})
```

this one works, but don't have to do it, like you want !